docs/index.rst

```diff
@@ -33,7 +33,6 @@ Get more information on how to meet the community, get involved, stay in touch.
 * Join our :doc:`Slack network </slack>` with thousands of other documentarians
 * Join a :doc:`local or online meetup </meetups/index>` to dive deeper into the community
 * Learn more about our :doc:`sponsorship options <sponsorship/index>` for your company
-* Fill out our `2024 Documentation Salary Survey <https://salary-survey.writethedocs.org/>`__ and `browse previous survey results  </surveys/>`__
```

Comment:
Don't wanna leave something like 'browse last years results while you wait for us to process 2024 or some such?

Reply:
Cos we're removing the link to the surveys entirely here.

Reply:
that's down in the next section anyway...